src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -180,7 +181,7 @@ class CrossDeviceLinkUI extends Component {
       <div>
         { error.type ?
           <SmsError error={error} trackScreen={this.props.trackScreen}/> :
-          <h1 className={`${theme.title} ${style.title}`}>Continue verification on your mobile</h1> }
+          <Title title='Continue verification on your mobile' /> }
```

Comment:
How come none of the `/crossDevice/` strings are in a strings file?

Reply:
They will be when we do localisation. This PR is mostly about making the copy specific to the document you selected so the cross device strings weren't affected.